### DISCUSSION TITLE
GotoTag Equivalent External Command

### DISCUSSION THREAD

**Discussion by SunnyOd** [leftwm/leftwm#725] [Q&A] [answered]
Hi,

Trying to get sxhkd working using latest 0.3.0 release of LeftWM. Is there an equivalent External command for GotoTag? SendWorkSpaceToTag doesn't seem to do what I expect and takes 2 arguments instead of the expected 1

Thanks!
Sunny

**Comment by VuiMuich** [comment, marked as answer]
Welcome adventurer!
This is actually the equivalent command. I am not sure why the external command has the workspace parameter actually.
I think we already had a discussion to unify this along with other planned breaking changes.

Back when this command was added, compatibility to external hotkey daemons was probably not a priority. You could have a look, how the `basic_polybar` uses it to make its tag buttons clickable. I am not experienced enough with sxhkd to tell how a script to determine the focused workspace could be used, though...
